components/page-templates/MyAccountTemplate/MyAccountTemplate.tsx

```diff
@@ -201,39 +235,35 @@ const MyAccountTemplate = () => {
             <Box key={data.id}>
               <Accordion disableGutters sx={{ ...style.accordion }}>
                 <AccordionSummary
-                  expandIcon={<ExpandMoreIcon sx={{ ...style.expandedIcon }} />}
+                  onClick={() => data.path && router.push(data.path)}
+                  expandIcon={data.component && <ExpandMoreIcon sx={{ ...style.expandedIcon }} />}
                   aria-controls={data.controls}
                   id={data.id}
                   sx={{ ...style.accordionSummary }}
                 >
                   <Typography variant="h3">{data.header}</Typography>
                 </AccordionSummary>
-                <AccordionDetails>{data.component}</AccordionDetails>
+                {data.component && <AccordionDetails>{data.component}</AccordionDetails>}
               </Accordion>
               <Divider sx={{ borderColor: 'grey.500' }} />
             </Box>
           )
         })}
 
         <Box sx={{ ...style.myAccountChildren }}>
-          <Typography variant={mdScreen ? 'h1' : 'h2'}>{t('order-details')}</Typography>
+          <Typography variant={mdScreen ? 'h1' : 'h2'}>{t('orders')}</Typography>
         </Box>
 
-        {/* code for subscription below */}
         <Divider sx={{ borderColor: 'grey.500' }} />
-        {isSubscriptionEnabled && (
```

Comment:
We have to keep the existing code.

Reply:
@kibo-chandradeeptalaha In the new design we don't have a subscription link

Reply:
Done